metrics/exporter.go

```diff
@@ -126,11 +131,11 @@ func newMetricsExporter(config *metricsConfig, logger *zap.SugaredLogger) (view.
 	// If there is a Prometheus Exporter server running, stop it.
 	resetCurPromSrv()
 
-	if ce != nil {
-		// UnregisterExporter is idempotent and it can be called multiple times for the same exporter
-		// without side effects.
-		view.UnregisterExporter(ce)
+	// TODO: Move Stackdriver and Promethus operations before stoping to an interface.
```

Comment:
Could you file an issue for this and put the Issue ID here? Otherwise, it could be a very long time before someone touches this code again and notices it.

Reply:
Done.